contracts/interfaces/IEntryPoint.sol

```diff
@@ -100,14 +100,15 @@ interface IEntryPoint is IStakeManager {
      *          perform the equivalent check using an off-chain library code
      * @return preOpGas total gas used by validation (including contract creation)
      * @return prefund the amount the wallet had to prefund (zero in case a paymaster pays)
+     * @return paymasterStake if current UserOperation uses a paymaster, then returns the current stake of this paymaster
```

Comment:
Maybe mention that the bundler should check if it meets its minimum requirement, and drop the op if it doesn't.